libs/csm/rest/csm_rest_query_deployment.py

```diff
@@ -47,7 +47,7 @@ def get_solution_yaml(self):
         solution_path = self.master.copy_file_to_local(remote_path=remote_sol_path,
                                                                local_path=local_sol_path)
         self.log.info(solution_path)
-        with open(local_sol_path, 'r') as yaml_file:
+        with open(local_sol_path, 'rb') as yaml_file:
```

Comment:
Can resolve pylint warning and remove format string from this file?

Reply:
after this fix, warning is not seen after running pylint and in codacy

Reply:
![image](https://user-images.githubusercontent.com/64882639/188409170-33e7ed60-b3ac-47dc-a55a-c4fe6483860d.png)

Reply:
Only 1 one is resolved. Can u resolve other 4 too?

Reply:
resolved.